.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -696,21 +696,21 @@ private bool SkipNonDecimalSeparator(char ch, int distance)
             return ch == Config.NonDecimalSeparatorChar && !(distance <= decimalLength && isMultiDecimalSeparatorCulture);
         }
 
-        private List<string> GetMatches(string input)
-        {
-            var successMatch = TextNumberRegex.Match(input);
-            var matchStrs = new List<string>();
-
-            // Store all match str.
-            while (successMatch.Success)
-            {
-                var matchStr = successMatch.Groups[0].Value;
-                matchStrs.Add(matchStr);
-                successMatch = successMatch.NextMatch();
-            }
-
-            return matchStrs;
-        }
+        // private List<string> GetMatches(string input)
+        // {
+        //    var successMatch = TextNumberRegex.Match(input);
+        //    var matchStrs = new List<string>();
+
+        // // Store all match str.
+        //    while (successMatch.Success)
+        //    {
+        //        var matchStr = successMatch.Groups[0].Value;
+        //        matchStrs.Add(matchStr);
+        //        successMatch = successMatch.NextMatch();
+        //    }
+
+        // return matchStrs;
+        // }

```

Comment:
Why is this method commented?

Reply:
I didn't suppose to let that commented, but I forgot. I already removed it, thanks for the feedback!